### ISSUE TITLE
💡Idea... more UI components?

### ISSUE THREAD

**Issue by vitalijalbu** [mantinedev/ui.mantine.dev#59]
Hi all, would be fine if members of mantine community can submit thir own custom components to mantine UI... just to increase the components count and show how powerful mantine is.... @rtivital what do you think?

**Comment by rtivital** [comment]
I'm against it: you add component once and your job is done, then I'll need to maintain it for the rest of my life.

**Comment by vitalijalbu** [comment]
> I'm against it: you add component once and your job is done, then I'll need to maintain it for the rest of my life.

Yes right, we can surely suggest components with code (that don't need to be updated in future or maintained) and that's all.

**Comment by iam-Akshat** [comment]
@rtivital Would it be a good idea to have a section of components powered by community, a seperate section on the website, can come with a disclaimer `May not be maintained or outdated` along with some metadata like when the last PR was made for that component.

**Comment by AKRking** [comment]
> @rtivital Would it be a good idea to have a section of components powered by community, a seperate section on the website, can come with a disclaimer `May not be maintained or outdated` along with some metadata like when the last PR was made for that component.

that what i was thinking,with large community contribution mantine will be more useful and popular

**Comment by vitalijalbu** [comment]
> I'm against it: you add component once and your job is done, then I'll need to maintain it for the rest of my life.

we can create PR and you jsut commit the code, no need you maintain all community components

**Comment by shlomo20** [comment]
> I'm against it: you add component once and your job is done, then I'll need to maintain it for the rest of my life.

I would Like to add to the point that others are rising, why not have something like https://uiverse.io/? it does not need to be so advanced that everyone can build from the FE could be through GitHub, the point being that it's clear that the components are maintained by end users,

the benefits to consider 
- would help showcase how people utilize the library  in different ways
- would help with inspiration
- would help to see different ideas and approaches that you would never have thought of by yourself
- would ease user adaptability 

thank you for considering this

**Comment by mateuscqueiros** [comment]
@shlomo20 Isn't that what [Mantine UI](https://ui.mantine.dev/) is tho? If that's the case then would be a good idea imo for people to add new components there, I'm sure everyone has a ton of components they made themselves using Mantine and would like to showcase. Maybe a community tab?

**Comment by jamalsoueidan** [comment]
I would love that our community build more ui examples on the ui.mantine

Maybe a calendar example with indicators that show some events happening on that day.

**Comment by andrewstec** [comment]
I have to agree. When I look at the issues, I see a request for a scheduler component. Your request sounds quite similar. 

If there are low-maintenance options to contribute components, it would be very beneficial. My googles and ChatGPT searches are running dry. Even ChatGPT has trouble creating components because it needs more examples out there.

> I would love that our community build more ui examples on the ui.mantine
> 
> Maybe a calendar example with indicators that show some events happening on that day.
